### ISSUE TITLE
Scatter plot markers disappear completely when using facecolors='none'

### ISSUE THREAD

**Issue by ThimoFerede** [matplotlib/matplotlib#25764]
Code:
```
import matplotlib.pyplot as plt
import numpy as np
plt.rcParams['figure.facecolor'] = 'w'
plt.scatter(np.random.randn(100), np.random.randn(100), facecolors='none', marker='o')
```

Expected result:
![image](https://user-images.githubusercontent.com/38126472/234114458-b3fecc78-2cae-4328-a665-5e49e3b87200.png)

Result:
![d54fea29-dfbc-4663-ba4c-8339c46e4130](https://user-images.githubusercontent.com/38126472/234114493-106a5a27-bd2c-4230-a3a6-ffcc533248f6.png)

Matplotlib version: 3.7.1
Python version: 3.10.10
Running in Anaconda3 environment using VS code

**Comment by jklymak** [comment]
`scatter.edgecolors='face'` is the default, and you've set the facecolor to 'none'.  If you want edges, you should specify *edgecolors*.

**Comment by ThimoFerede** [comment]
I wanted to use this with c and cmap and not a constant color for the edges. Is that still possible?

**Comment by ksunden** [comment]
#25593 is intending to fix that usecase and #24404 has an (admittedly inelegant) workaround that works with current mpl
